api/middleware/applicationValidation.js

```diff
@@ -0,0 +1,12 @@
+const validation = (schema) => async (req, res, next) => {
```

Comment:
This would work great as a general yup validation middleware. If we think we might want yup validation elsewhere, consider moving this / renaming this outside of just the application endpoints.

Reply:
Hey Jacob, I branched of my yup validation middleware and so those commits are showing up here. If you want to review that branch: https://github.com/BloomTech-Labs/underdog-devs-be-a/pull/60